.github/workflows/terraform_destroy_on_delete.yaml

```diff
@@ -4,7 +4,7 @@ on: [delete]
 
 jobs:
   build:
-    if: contains(github.event.ref_type, 'branch') &&  (! github.event.ref == 'master') && (! github.event.ref == 'develop')
+    if: contains(github.event.ref_type, 'branch') &&  (! contains(github.event.ref, 'master')) && (! contains(github.event.ref, 'develop'))
```

Comment:
Well, this would mean that branches with "master" or "develop" in their names would not be deleted (which was the original problem). It looks like the original has two problems: The leading exclamation mark is not the right way to specify inequality, and we're comparing to develop and master whereas I think in this case we should be comparing to the full path. How about changing to this line:
if: github.event.ref_type == 'branch' && (github.event.ref != 'refs/heads/master') && (github.event.ref != 'refs/heads/develop')

Reply:
OK, thanks, made that change, and it makes sense to me now [like why we were using contains before, but it got it wrong for merge_from_master].   So, if you just want to approve this, I'll check this in and try testing it.